src/test/ui/label/label_misspelled.rs

```diff
@@ -0,0 +1,18 @@
+fn main() {
+    'LOOP: loop {
+        LOOP;
+        //~^ ERROR cannot find value `LOOP` in this scope
+    };
+    'while_loop: while true { //~ WARN denote infinite loops with
```

Comment:
Just wondering, is the recommended label capitalization `SCREAMING_SNAKE_CASE` or `snake_case`?

Reply:
the recommended label is `snake_case` from what I know